components/utils/locale.js

```diff
@@ -1,7 +1,12 @@
 const {supportedLocales, fallbackLocale} = require('../../config.json')
 
-let {LOCALE: lang} = process.env
-if (!supportedLocales.includes(lang)) {
-  lang = fallbackLocale
+function locale() {
+  const {LOCALE: lang} = process.env
+  if (supportedLocales.includes(lang)) {
+    return lang
+  }
+
+  return fallbackLocale
 }
-module.exports = lang
+
+module.exports = () => locale()
```

Comment:
Because when we're running our tests we're setting `process.env.LOCALE` a couple of times and it will keep using the first set value in our component, which will cause our tests to fail.
At least that is what I'm thinking...

Maybe there is a better way of doing this, but I don't know how :thinking: .